README.md

```diff
@@ -1,93 +1,67 @@
 
+### Commands
 ```
-$ tag-release
-
-? What is your GitHub username
-? What is your GitHub password
-? What type of release is this (Use arrow keys)
-❯ Major (Breaking Change)
-  Minor (New Feature)
-  Patch (Bug Fix)
-etc...
+$ tag-release --help
 ```
 
-### With Flags
+Will display a list of avaliable commands and options.
 
 ```
-$ tag-release --help
-$ tag-release -h
-
-Usage: tag-release [options]
-
-Options:
-
-  -h, --help                     Output usage information
-  -c, --config [filePath]        Path to JSON Configuration file (defaults to './package.json')
-  -r, --release [type]           Release type (major, minor, patch)
-  --verbose                      Console additional information
-  -V, --version                  Output the version number
-  -p, --prerelease               Create a pre-release
-  -i, --identifier <identifier>  Identifier used for pre-release
-  --reset                        Reset repo to upstream master/develop branches
-  --promote [tag]                Promotes specified pre-release tag to an offical release
-  --continue                     Continues the rebase process of a tag promotion
-  --qa [scope]                   Create initial upstream branch for upstream repo
-  --pr [scope]                   Update package.json to bump upstream repos and create a PR to develop
-  --maxbuffer <n>                Overrides the max stdout buffer of the child process. Size is 1024 * <n>.
-
-Examples:
-
-   $ tag-release
-   $ tag-release --config ../../config.json
-   $ tag-release -c ./manifest.json
-   $ tag-release --release major
-   $ tag-release -r minor
-   $ tag-release --verbose
-   $ tag-release -V
-   $ tag-release --version
-   $ tag-release --prerelease
-   $ tag-release -p
-   $ tag-release -p -i foo
-   $ tag-release --reset
-   $ tag-release --promote
-   $ tag-release --promote v1.1.1-my-tagged-version.0
-   $ tag-release --continue
-   $ tag-release --qa
-   $ tag-release --qa myorg
-   $ tag-release --pr
-   $ tag-release --pr myorg
-   $ tag-release --maxbuffer 5000
+  Usage: tag-release [options] [command]
+
+
+  Options:
+
+    -V, --version  output the version number
+    -h, --help     output usage information
+
+
+  Commands:
+
+    continue                     continue from a previously conflicted state
+    config <filePath>            override .json configuration file path, defaults to './package.json'
+    dev                          create a PR from origin feature branch to upstream feature branch
+    pr [scope]                   update consumer project feature branch and create a PR to develop
+    prerelease|pre [identifier]  create a pre-release
+    promote|pro [tag]            promote a pre-release tag previously created by tag-release
+    qa [scope]                   create initial upstream feature branch for consumer project
+    reset                        reset repo to upstream master/develop branches
+    help [cmd]                   display help for [cmd]
```

Comment:
Where is `start` listed?

Reply:
I made `start` not show help because I don't want to be know about it/run it. Just use `tag-release` and that is the default flow. =)